virt-who-config.5

```diff
@@ -23,7 +23,7 @@ The following are options that can be specified in the \fBglobal\fR section:
 how often to check connected hypervisors for changes (seconds). Also affects how often a mapping is reported.
 .TP
 \fBreporter_id\fR
-The id of this virt-who instance, reported with all mappings.
```

Comment:
These options will still be allowable in the configuration files.
The man pages changes necessary are in ./virt-who.8

Please update that man page accordingly.

Thank you!!